brianlong/www.validators.app
# 180458988_add_stake_pools_fees

#### What's this PR do?
-  add stake pools fees

#### How should this be manually tested?
- Run migration
- run `rake add_stake_pool:mainnet `
- run `rake update_fee_in_stake_pools:mainnet`
- Check in console if `manager_fee` has been filled in `stake_pools` table

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/180458988

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [Y] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
